app/model.go

```diff
@@ -37,19 +41,26 @@ func InitialModel() *Model {
 	})
 
 	return &Model{
-		viewport:     vp,
-		currentView:  loading,
-		systemInfo:   systeminfoview.New(version),
-		viewStack:    viewstack.Stack{},
-		commandInput: cmdBar(),
+		viewport:       vp,
+		currentView:    loading,
+		systemInfo:     systeminfoview.New(version),
+		viewStack:      viewstack.Stack{},
+		commandInput:   cmdBar(),
+		terminalWidth:  80,
+		terminalHeight: 20,
 	}
 }
 
 // Init  will be automatically called by Bubble Tea if the model implements the Model interface
 // and is passed into the tea.NewProgram function.
 func (m *Model) Init() tea.Cmd {
 	// "" loads all stacks on all nodes
-	return tea.Batch(tick(), loadSnapshotAsync(), systeminfoview.LoadStatus())
+	return tea.Batch(
+		tick(), 
+		loadSnapshotAsync(), 
+		systeminfoview.LoadStatus(),
+		m.systemInfo.Init(), // Initialize systeminfo's tick commands
```

Comment:
We call 2 times into system view here. If load status is needed in the initialization process, it should be called from within `Init()`.

Reply:
@clangenb please fix/suggest or create a follow-up issue

Reply:
https://github.com/mosonyi/swarmcli/issues/113